api_core/tests/unit/test_page_iterator.py

```diff
@@ -69,6 +70,18 @@ def test_iterator_calls_parent_item_to_value(self):
         item_to_value.assert_called_with(parent, 12)
         assert page.remaining == 97
 
+    def test_raw_page(self):
+        parent = mock.sentinel.parent
+        item_to_value = mock.sentinel.item_to_value
+
+        raw_page = mock.sentinel.raw_page
+
+        page = page_iterator.Page(parent, (1, 2, 3), item_to_value, raw_page=raw_page)
+        assert page.raw_page is raw_page
+
+        self.assertRaises(AttributeError):
```

Comment:
```suggestion
        with self.assertRaises(AttributeError):
```

Reply:
@software-dov I don't know why, but I'm not allowed to commit that change.

Reply:
Woah, just realized what I did here. This slipped through because I gave up trying to run unit tests locally after spending two hours fighting with virtualenv and figured CI would catch any errors. Fixed.

Reply:
We normally run tests using [`nox`](http://nox.thea.codes/en/stable/), which handles setting up the virtualenvs correctly for each kind of test.